internal/api/db/handlers/review.go

```diff
@@ -0,0 +1,82 @@
+// follow https://cgrant.medium.com/developing-a-simple-crud-api-with-go-gin-and-gorm-df87d98e6ed1
+
+package handlers
```

Comment:
I would move this to `internal/api/handlers` and not nest it under the `db` folder, since it's not necessarily a concern of the database.